styleguide/screens/Patterns/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import dedent from 'dedent';
+
+import Dropdown, { HORIZONTAL_ATTACHMENTS } from '../../../../components/Dropdown/Dropdown';
+import Specimen from '../../../components/Specimen/Specimen';
+import { H, T, D, A, C, Bq, List, Placeholder } from '../../../components/Scaffold/Scaffold';
+
+import m from '../../../../globals/modifiers.css';
+import css from './Dropdown.css';
+
+// eslint-disable-next-line react/prop-types
+const Target = ({ onClick, placeholderText }) => (
+  <Placeholder onClick={ onClick }>{ placeholderText }</Placeholder>
+);
+
+export default class DropdownDocumentation extends Component {
+  state = {
+    showDropdown: false,
+  };
+
+  handleClick = () => {
+    this.setState(({ showDropdown }) => ({
+      showDropdown: !showDropdown,
+    }));
+  };
+
+  render() {
+    const { showDropdown } = this.state;
+
+    return (
+      <div>
+        <H level={ 1 }>Dropdown</H>
+        <T elm="p" className={ cx(m.mtr, m.largeI, m.demi) }>
+          Dropdowns are toggleable overlays. They’re typically used to add additional information or
+          functionality to a given element. Only trigger the display of a dropdown using
+          a <C>click</C> or <C>hover</C>.
+        </T>
+        <D>
+          <T elm="p" className={ m.mtr }>
+            For consistency’s sake, all dropdowns also wrap their content in <C>DropdownInner</C>
+          </T>
+          <T elm="p" className={ m.mtr }>
+            Attach it to a specific element, by proving that element in the <C>target</C> prop
+          </T>
+          <Specimen
+            classNames={ {
+              root: m.mtr,
+              specimenContainer: css.specimenContainer,
+            } }
+            code={ dedent`
+            <Dropdown
+              horizontalAttachment={ HORIZONTAL_ATTACHMENTS.LEFT }
+              target={ <Target /> }
+              flushHorizontal
+              active={ this.state.showDropdown }
+            >
+              <DropdownContent />
+            </Dropdown>
+          ` }
+          >
+            <Dropdown
+              horizontalAttachment={ HORIZONTAL_ATTACHMENTS.LEFT }
+              target={
+                <Target
+                  onClick={ this.handleClick }
+                  placeholderText={ !showDropdown ? 'Open dropdown' : 'Close dropdown' }
+                />
+              }
+              flushHorizontal
+              active={ showDropdown }
+              targetClassName={ css.target }
+            >
+              <Placeholder onClick={ this.handleClick }>Close dropdown</Placeholder>
```

Comment:
A place holder in the dropdown looks really weird. I think we should reserve the use of it specifically for being the top level component in specimen's child prop.

Noticed a bug in the dropdown component too, where you can close it by clicking outside the component yet it doesn't update the state. We should probably look at fixing that!